<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(248, 55, 55);
border-radius: 12px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>0</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Youtube&lt;/strong&gt; to &lt;strong&gt;MP3/MP4&lt;/strong&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_choose_filetype">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>371</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Choose the one you would like to translate:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxFileType">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 184, 185);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_url_youtube">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string> URL of the Youtube video:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEditUrlYoutube">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 178, 179);</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFileSize">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string> Choose the size of the file:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidgetLegend">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>221</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 178, 179);</string>
       </property>
       <item>
        <property name="text">
         <string>'Light' =&gt; Poor image resolution but lightweight file </string>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </item>
       <item>
        <property name="text">
         <string>'Medium' =&gt; Medium image resolution and mediumweight file </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>'Better' =&gt; Best image resolution and largeweight file</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBoxLight">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>160</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Light</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxMedium">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>200</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Medium</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxBetter">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>240</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Better</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFileName">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>280</y>
         <width>231</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>If you would like to name the file:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEditFileName">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>290</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 178, 179);</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelLegendNameDefault">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 178, 179);</string>
       </property>
       <property name="text">
        <string>Else, it would name as the title of the video</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBarMain">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>191</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar{
	background-color: rgb(255, 165, 167);
	color: rgb(255, 255, 255);
	border-style: none;
	border-radius: 12px;
	text-align: center;
}

QProgressBar::chunk{
	border-radius: 12px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.557, x2:1, y2:0.557, stop:0 rgba(170, 55, 56, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSubmit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>330</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 178, 179);
border: 1px solid black;
</string>
       </property>
       <property name="text">
        <string>SUBMIT</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelLoadingDownload">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>390</y>
         <width>381</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfos">
    <property name="title">
     <string>Infos</string>
    </property>
    <addaction name="actionLink_to_github"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuInfos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLink_to_github">
   <property name="text">
    <string>Link to github</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
